test/utils.go

```diff
@@ -9,25 +9,19 @@ import (
 	"testing"
 
 	ft "github.com/TRON-US/go-unixfs"
+	balanced "github.com/TRON-US/go-unixfs/importer/balanced"
 	h "github.com/TRON-US/go-unixfs/importer/helpers"
 	trickle "github.com/TRON-US/go-unixfs/importer/trickle"
 
-	cid "github.com/ipfs/go-cid"
 	chunker "github.com/TRON-US/go-btfs-chunker"
+	cid "github.com/ipfs/go-cid"
 	u "github.com/ipfs/go-ipfs-util"
 	ipld "github.com/ipfs/go-ipld-format"
 	mdag "github.com/ipfs/go-merkledag"
 	mdagmock "github.com/ipfs/go-merkledag/test"
 	mh "github.com/multiformats/go-multihash"
 )
 
-// SizeSplitterGen creates a generator.
-func SizeSplitterGen(size int64) chunker.SplitterGen {
```

Comment:
Have existing unit tests in this go-unixfs passed without an error? Thanks.

Reply:
yes. I think this is left over dup from ipfs